exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/spill/SpillSet.java

```diff
@@ -419,6 +462,13 @@ public SpillSet(DrillConfig config, FragmentHandle handle, PhysicalOperator popC
         operName, handle.getMajorFragmentId(), popConfig.getOperatorId(), handle.getMinorFragmentId());
   }
 
+  @VisibleForTesting
+  public SpillSet(String baseDir) {
```

Comment:
We're trying to use the Hadoop `Path` class as our preferred way to pass file and directory names around.